.github/CODEOWNERS

```diff
@@ -1,3 +1,20 @@
 # Enforces admin protections for repo configuration via probot settings app.
 # ref: https://github.com/probot/settings#security-implications
+
+# Maintainers
+# - Stephen Augustus (@justaugustus)
+# - Jason DeTiberus (@detiber)
+# - Márk Sági-Kazár (@sagikazarmark)
+# - Ed Warnicke (@edwarnicke)
+# - Shannon McFarland (@shmcfarl)
+# - Lelia Bray-Musso (@lelia)
+# - Sam Betts (@sambetts-cisco)
```

Comment:
Plus @svrnm

Reply:
O right, he wasn't yet an org member when I filed this :D